scripts/ansible/roles/linux/intel/tasks/sgx-driver.yml

```diff
@@ -29,11 +29,21 @@
 
 - name: Download Intel SGX DCAP Driver
   get_url:
-    url: "{{ intel_sgx_driver_url }}"
+    url: "{{intel_sgx_w_flc_driver_url}}"
     dest: /tmp/sgx_linux_x64_driver_dcap.bin
     mode: 0755
     timeout: 120
   retries: 3
+  when: flc_enabled|bool
```

Comment:
I don't believe there is a need to use the Jinja filter `|bool` here. 

I would expect the system to know that this is a boolean value, more can be read https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html.